csi-plug-n-play/sidecars/soda-csi-provisioner/external-provisioner/pkg/controller/controller.go

```diff
@@ -427,6 +429,55 @@ func (p *csiProvisioner) ProvisionExt(options controller.ProvisionOptions) (*v1.
 		return nil, controller.ProvisioningFinished, errors.New("storage class was nil")
 	}
 
+	//SODA-CHANGE
+	backendDriverName, err := GetDriverName(p.grpcClient, operationTimeout)
+	if err != nil {
+		klog.Fatalf("Error getting CSI driver name: %s", err)
+	}
+	klog.Infof("The Backend Driver Name is : %s ",backendDriverName)
+	klog.Infof("The provisioner.DriverName  is : %s ",p.driverName)
+
+	//TODO Make this code work with SODA API-Server (Issue is with Grpc version, soda uses v1.29.1 whereas csi-provisioner uses v1.26.0
```

Comment:
Instead of merging commneted code, we can merge it after resolving

Reply:
Let's keep this code or else we need to maintaine this code seperately.